tasks/robosignatory.yml

```diff
@@ -0,0 +1,69 @@
+# tasks for role robosignatory setup
+---
+- name: Install robosignatory packages
+  yum:
+    name:
+    - python3-robosignatory
+    - fedora-messaging
+    - trousers
+    - tpm-tools
+    - sigul
```

Comment:
Is there a hard requirement on sigul somewhere in our dependency chain? if not let's remove this

Reply:
> Is there a hard requirement on sigul somewhere in our dependency chain? if not let's remove this

No. It was a vestige from the Fedora role. I added a commit to drop it.

There's also nothing in the role that currently demands `trousers` or `tpm-tools`. Should they be dropped as well?

In the process of dropping `sigul`, I also discovered that the `buildtools` repo isn't needed--so I dropped the enablement of that. And, I also moved the installation of the `fedora-messaging` package to `tasks/fedora-messaging.yml` instead of `tasks/robosignatory.yml`.